resources/views/install/requirements.blade.php

```diff
@@ -11,7 +11,7 @@
 		<li class="list__item list__title {{ $phpSupportInfo['supported'] ? 'success' : 'error' }}">
 		<strong>{{ ucfirst($type) }}</strong>
 		@if ($type == 'php')
-			<strong><small>(version {{ $phpSupportInfo['minimum'] }} required)</small></strong>
+			<strong><small>(version {{ $phpSupportInfo['minimum'] }} minimum required)</small></strong>
```

Comment:
Sure.
or 
```suggestion
			<strong><small>(version {{ $phpSupportInfo['minimum'] }} or higher required)</small></strong>
```
:man_shrugging:

Reply:
Even better!